ontology/yaml/resources/HVAC/entity_types/AHU.yaml

```diff
@@ -2667,6 +2667,230 @@ AHU_BYPSSPC_DX3SC_ECON_HT2SC_SFSS_SFVSC:
+  is_canonical: true
+  implements:
+  - AHU
+  - UV
+  - H3X
+  - DAM
+  - DFCM
+  - CHWVM
+  - DFSS
+  opt_uses:
+  - control_status
+  - run_command
+  - run_status
+  - lost_power_alarm
+  - filter_alarm
+  - control_mode
+  - run_mode
+  - schedule_run_command
+
+AHU_DFSS_DAM_DFCM_CHWVM_RTM:
+  description: "AHU with discharge air/fan and chilled water valve operation. "
+  is_canonical: true
+  implements:
+  - AHU
+  - DFSS
+  - DAM
+  - DFCM
+  - CHWVM
+  - RTM
+  opt_uses:
+  - control_status
+  - lost_power_alarm
+  - filter_alarm
+  - control_mode
+  - run_mode
+  - schedule_run_command
+  - smoke_alarm
+
+AHU_DFSS_DAM_DFCM_CHWVM_RTM_ELCTS:
+  description: "AHU with discharge air/fan and chilled water valve operation. Leaving and entering cooling coil temperature are monitored."
+  is_canonical: true
+  implements:
+  - AHU
+  - DFSS
+  - DAM
+  - DFCM
+  - CHWVM
+  - RTM
+  uses:
+  - leaving_cooling_coil_temperature_sensor
+  - entering_cooling_coil_temperature_sensor
+  opt_uses:
+  - control_status
+  - lost_power_alarm
+  - filter_alarm
+  - control_mode
+  - run_mode
+  - schedule_run_command
+  - smoke_alarm
+
+AHU_DFSS_DAM_DFCM_CHWVM_RTM_LCTS:
+  description: "AHU with discharge air/fan and chilled water valve operation. Leaving cooling coil temperature is monitored."
+  is_canonical: true
+  implements:
+  - AHU
+  - DFSS
+  - DAM
+  - DFCM
+  - CHWVM
+  - RTM
+  uses:
+  - leaving_cooling_coil_temperature_sensor
+  opt_uses:
+  - control_status
+  - lost_power_alarm
+  - filter_alarm
+  - control_mode
+  - run_mode
+  - schedule_run_command
+  - smoke_alarm
+
+AHU_DFSS_DAM_CHWVM_RTM:
+  description: "AHU with discharge air and chilled water valve operation."
+  is_canonical: true
+  implements:
+  - AHU
+  - DFSS
+  - DAM
+  - CHWVM
+  - RTM
+  opt_uses:
+  - lost_power_alarm
+  - filter_alarm
+  - control_mode
+  - run_mode
+  - schedule_run_command
+  - smoke_alarm
+  - entering_cooling_coil_temperature_sensor
+  - leaving_cooling_coil_temperature_sensor
+  - control_status
+
+AHU_CHWDT_CHWVM_CLPM_DFSS_ZTM:
+  description: "AHU with chilled water valve operation and discharge fan operation. Chilled return/supply temp and zone air temp monitoring."
+  is_canonical: true
+  implements:
+  - AHU
+  - CHWDT
+  - CHWVM
+  - CLPM
+  - DFSS
+  - ZTM
+  uses:
+  - chilled_water_valve_percentage_command
+  opt_uses:
+  - filter_differential_pressure_status
+  - run_command
+  - run_status
+  - lost_power_alarm
+  - filter_alarm
+  - control_mode
+  - run_mode
+  - schedule_run_command
+  - smoke_alarm
+  - control_status
+
+AHU_DFVSC_CHWVM_CLPM_DTC_UV_CO2C_MTM_OAFM_RHM_CHWDT:
+  description: "AHU with discharge air, chilled water valve, discharge air/fan, CO2, UV and chilled supply/return water operation. Outside air monitoring"
+  is_canonical: true
+  implements:
+  - AHU
+  - DFVSC
+  - CHWVM
+  - CLPM
+  - DTC
+  - UV
+  - CO2C
+  - MTM
+  - OAFM
+  - RHM
+  - CHWDT
+  - DFSS
+  uses:
+  - return_air_temperature_setpoint
```

Comment:
should be from implementation

Reply:
updated to AHU_CHWDT_CHWVM_CLPM_DFSS_CHWZTC. The zone_air_temperature_setpoint is missing.